app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -385,12 +389,13 @@ function AcuantCapture(
    */
   function startCaptureOrTriggerUpload(event) {
     if (event.target === inputRef.current) {
-      const shouldStartEnvironmentCapture =
-        hasCapture && capture !== 'user' && !isForceUploading.current;
+      const isAcuantCaptureCapable = hasCapture && !acuantFailureCookie;
+      const shouldStartAcuantCapture =
+        isAcuantCaptureCapable && capture !== 'user' && !isForceUploading.current;
       const shouldStartSelfieCapture =
         isAcuantLoaded && capture === 'user' && !isForceUploading.current;
```

Comment:
CodeClimate is unhappy about something here: https://codeclimate.com/github/18F/identity-idp/pull/5590
> Consider simplifying this complex logical expression. NEW

Reply:
> CodeClimate is unhappy about something here: https://codeclimate.com/github/18F/identity-idp/pull/5590
> 
> > Consider simplifying this complex logical expression. NEW

Yeah, I noticed this in https://github.com/18F/identity-idp/pull/5590#issuecomment-964317359 . I tried successfully to appease it with ce805ea, but no luck 😞

Reply:
Ah, I missed that comment. I'm not sure what tool CodeClimate is using for JS complexity (for per-line or per-section), so I'm not sure what the right way to ignore it is, except ignoring the whole file: https://github.com/18F/identity-idp/blob/c7ec3ac85eae389a4eff5d55270b9a07bad7ee29/.codeclimate.yml#L56-L57

Another idea might be to break out a few helper methods? `isAcuantCaptureCapable() && isUserCapture() && isNotForceUploading()` or something?

But yes bottom line, I'd be in favor of ignoring this file to get around this

Reply:
How'd you feel about disabling the CodeClimate check altogether?

```diff
diff --git a/.codeclimate.yml b/.codeclimate.yml
index 02dc06e21..a98b2f902 100644
--- a/.codeclimate.yml
+++ b/.codeclimate.yml
@@ -4,8 +4,7 @@ checks:
     config:
       threshold: 4
   complex-logic:
-    config:
-      threshold: 4
+    enabled: false
   file-lines:
     enabled: false
   method-complexity:
```

Reply:
I'm fine with that

Reply:
> I'm fine with that

👍  Updated in 7cf8b6b.